src/pages/Dashboard/components/DashboardChart.tsx

```diff
@@ -0,0 +1,121 @@
+import type { ResponsiveValue } from '@chakra-ui/react'
+import {
+  Button,
+  Card,
+  CardHeader,
+  Flex,
+  Heading,
+  Skeleton,
+  Stat,
+  StatArrow,
+  StatNumber,
+  Switch,
+} from '@chakra-ui/react'
+import type { HistoryTimeframe } from '@shapeshiftoss/types'
+import type { Property } from 'csstype'
+import { useCallback, useState } from 'react'
+import { Amount } from 'components/Amount/Amount'
+import { BalanceChart } from 'components/BalanceChart/BalanceChart'
+import { TimeControls } from 'components/Graph/TimeControls'
+import { MaybeChartUnavailable } from 'components/MaybeChartUnavailable'
+import { Text } from 'components/Text'
+import { useTimeframeChange } from 'hooks/useTimeframeChange/useTimeframeChange'
+import {
+  selectChartTimeframe,
+  selectPortfolioAssetIds,
+  selectPortfolioLoading,
+  selectPortfolioTotalUserCurrencyBalanceExcludeEarnDupes,
+} from 'state/slices/selectors'
+import { useAppSelector } from 'state/store'
+
+const justifyContentMdSpaceBetween = { base: 'center', md: 'space-between' }
+const textAlignMdInherit: ResponsiveValue<Property.TextAlign> = { base: 'center', md: 'inherit' }
+const flexDirMdRow: ResponsiveValue<Property.FlexDirection> = { base: 'column', md: 'row' }
+const displayMdBlock = { base: 'none', md: 'block' }
+const justifyContentMdFlexStart = { base: 'center', md: 'flex-start' }
+const displayMdNone = { base: 'block', md: 'none' }
+const displayMdFlex = { base: 'none', md: 'flex' }
+const timeControlsButtonGroupProps = {
+  display: 'flex',
+  width: 'full',
+  justifyContent: 'space-between',
+  px: 6,
+  py: 4,
+}
+
+export const DashboardChart = () => {
+  const userChartTimeframe = useAppSelector(selectChartTimeframe)
```

Comment:
Do we reset this between pages, since this is used in various components?

Reply:
@reallybeard question, technically not introduced by this PR but could mean the chart timeframe is cached in-between pages, which may or may not be an issue depending on the flow we want